app/main.py

```diff
@@ -1,2 +1,77 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+
+    load_factor = 2 / 3
+
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.length = 0
+        self.hash_table: list = [None] * self.capacity
+
+    def resize(self) -> None:
+        hash_table_to_resize = self.hash_table
+        self.capacity *= 2
+        self.hash_table: list = [None] * self.capacity
+        self.length = 0
+        for node in hash_table_to_resize:
+            if node:
+                key, value, hash_value = node
+                self.__setitem__(key, value)
+
+    def __setitem__(self, key: int | float | str | bool, value: Any) -> None:
```

Comment:
write `key: Hashable ` instead of `int | float | str | bool`

Reply:
Thank you for all the remarks. Updated already.